setup.py

```diff
@@ -30,6 +27,7 @@
     author="Daniel Lenski",
     author_email="dlenski@gmail.com",
     license='GPL v3 or later',
+    python_requires=">=3",
```

Comment:
Really glad this exists now :+1: … but now I'll have to update all my repos :dizzy_face: